docs/src/use/command-line-interface.md

```diff
@@ -591,6 +592,16 @@ For example, suppose a rule has a bug that causes it to report a false positive,
 npx eslint --report-unused-disable-directives file.js
 ```
 
+#### `--report-unused-disable-directives-severity`
+
+Same as [`--report-unused-disable-directives`](#--report-unused-disable-directives), but allows you to specify the severity level (`error`, `warn`, `off`) of the reported errors. Only one of these two options can be used at a time.
```

Comment:
We should specify Argument Type here, see for example [`--cache-strategy`](https://eslint.org/docs/latest/use/command-line-interface#--cache-strategy).

Reply:
Added.